src/memory_manager.rs

```diff
@@ -119,15 +119,15 @@ const HEADER_RESERVED_BYTES: usize = 32;
 /// --------------------------------------------------
 /// Bucket `MAX_NUM_BUCKETS`              ↕ 1 byte
 /// --------------------------------------------------
-/// Unallocated space
+/// Unallocated space                     ↕ 30'688 bytes
 /// -------------------------------------------------- <- Buckets (Page 1)
 /// Bucket 1                              ↕ 1024 pages
-/// -------------------------------------------------- <- Page 1025
+/// -------------------------------------------------- <- Page N + 1
```

Comment:
nit: I'd also replace the "1024 pages" above and below with `N pages` for consistency.